plugins/modules/foreman_architecture.py

```diff
@@ -82,7 +82,12 @@
     state: absent
 '''
 
-RETURN = ''' # '''
+RETURN = '''
+entity:
+  description: Final state of the affected Entity
+  returned: success
+  type: dict
```

Comment:
given it is really a dict with arrays of dicts, can we be more verbose here?
also see https://github.com/theforeman/foreman-ansible-modules/issues/583

Reply:
Do you mean that level of 'more verbose'?

Reply:
Yes. That looks good. Ans now you can't just copy&paste this to all modules :P